src/models/propulsion/FGRocket.cpp

```diff
@@ -82,8 +86,8 @@ FGRocket::FGRocket(FGFDMExec* exec, Element *el, int engine_number, struct Input
   std::stringstream strEngineNumber;
   strEngineNumber << EngineNumber;
 
-  auto PropertyManager = exec->GetPropertyManager();
-  bindmodel(PropertyManager.get()); // Bind model properties first, since they might be needed in functions.
+  FGPropertyManager* PropertyManager = exec->GetPropertyManager();
+  bindmodel(PropertyManager); // Bind model properties first, since they might be needed in functions.
```

Comment:
```suggestion
  auto PropertyManager = exec->GetPropertyManager();
  bindmodel(PropertyManager.get()); // Bind model properties first, since they might be needed in functions.
```